PSWebServiceLibrary.php

```diff
@@ -47,7 +47,7 @@ class PrestaShopWebservice
     /** @var string Minimal version of PrestaShop to use with this library */
     const psCompatibleVersionsMin = '1.4.0.0';
     /** @var string Maximal version of PrestaShop to use with this library */
-    const psCompatibleVersionsMax = '8.2.0';
+    const psCompatibleVersionsMax = '9.0.0';
```

Comment:
As we haven't tested this module on 9.0.0, maybe we should set 8.2.99 to the max for now 🤔
(In the future, I think that we can remove totally the max compatible version thing here 🤷‍♂️)

```suggestion
    const psCompatibleVersionsMax = '8.2.99';
```

Reply:
cc @tleon